### ISSUE TITLE
[feature] export jobs result in different formats

### ISSUE THREAD

**Issue by mlodic** [intelowlproject/IntelOwl#830]
We could add a button or something like that the user can leverage to download a Job result in different formats like:

* raw JSON
* PDF
* MISP
* STIX
* etc

**Comment by CypherpunkSamurai** [comment]
Hello Sir, I had a question, Is this meant only for the webui or do we need to add the output converter to overall api?

**Comment by mlodic** [comment]
The frontend does talk with the same API that can be used by the command line interface.

So, usually the steps should be:
* add support of this feature in the backend (Django API)
* leverage the new or modified API in the frontend (GUI)
* leverage the new or modified API in the CLI (pyintelowl)

ATM we are working on the complete rework of the GUI (should be ready for the GSoC) so we do not suggest to make changes there now

**Comment by devmrfitz** [comment]
@mlodic We need the endpoint(s) which can present [this](https://intelowl.readthedocs.io/en/latest/OpenAPI.html#get--api-jobs-id) data in the various formats, right?

**Comment by mlodic** [comment]
yeah

**Comment by iharshit009** [comment]
Hey @mlodic  I would like to work on this issue. I am new to this project, if looks fine can I take this up?

**Comment by mlodic** [comment]
yeah, you can try to work on this. Please create a branch from the `dev-v4` branch instead of develop because there we have the new GUI and you'll need to work on that.

This is a pretty complete task (frontend,backend,cli) so it is really a good way to touch most of the things in the project

**Comment by fear-the-reaper** [comment]
Can a provide a sample JSON output? I know STIXv2.1 since STIX is basically JSON I just need to map it thanks!

**Comment by mlodic** [comment]
JSON output may vary based on the analyzers requested. Mapping to STIX could be a really complex task to do. I think that this issue should be rethougth in a more feasible manner. Some export formats like STIX or MISP cannot just be converted so easily.

**Comment by mlodic** [comment]
closed because unclear the goal
